relationship.md

```diff
@@ -0,0 +1,56 @@
+# Relationship among WoT Groups
+
+This document explains how different WoT groups within W3C can/should be linked to each other.
+
+- WoT Working Group (WG): Responsible for writing normative and informative W3C Technical Reports (TRs), as well as establishing liaisons with related SDOs.
+- WoT Interest Group (IG): Responsible for doing official outreach, collecting use cases to extract requirements, and organizing plugfests.
+- Related Community Groups (CGs): They can participate in the activities of the WG or IG through well-defined mechanisms in addition to their own activities. See below for more details.
+
+There are relationships between those activities. E.g. use cases relate to TRs, outreach and plugfests relate to CGs who do outreach themselves. Per activity, a more detailed relationship is written below. 
+An overview figure is also available, which will be split to each activity later on.
+
+![Overview Image](./images/wot-groups-relationship.drawio.png)
+
+## Technical Report process
```

Comment:
"Technical report" sounds like the "Implementation report". Here, the WoT deliverables (REC and Notes) should be named here.

Reply:
The correct name is Technical Report but I will mention both.